src/UI/templates/default/Deck/deck.less

```diff
@@ -21,3 +21,23 @@
     }
   }
 }
+
+@media (min-width: @screen-sm-min) and (max-width: @screen-sm-max) {
+  .row-eq-height {
+
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    flex: 3;
```

Comment:
This is shorthand for flex-grow: 3;. Please change to enhance the readability.

why is this needed? why would any .row-eq-height need a separate flex-grow? If they all get grow 3, this would not have any effect, right, or am I missing something (which is very possible).